backend/pipelined/mutate.go

```diff
@@ -126,10 +126,9 @@ func (p *Pipelined) pipeMutator(mutator *types.Mutator, event *types.Event) ([]b
 
 	// Prepare log entry
 	fields := logrus.Fields{
-		"environment":  mutator.Environment,
-		"organization": mutator.Organization,
-		"mutator":      mutator.Name,
-		"assets":       mutator.RuntimeAssets,
+		"namespace": mutator.Namespace,
+		"mutator":   mutator.Name,
+		"assets":    mutator.RuntimeAssets,
```

Comment:
Looks like your branch got into a weird state, these lines do not exist on `devel/api` right now, which caused a conflict!